linuxkpi/gplv2/include/linux/interrupt.h

```diff
@@ -1,4 +1,12 @@
+#ifndef	__GPLV2_INTERRUPT_H__
+#define	__GPLV2_INTERRUPT_H__
+
 #include_next <linux/interrupt.h>
 
-/* LKPI_TODO */
-#define	tasklet_disable_nosync(x) 0
+static inline void tasklet_disable_nosync(struct tasklet_struct *t)
```

Comment:
This should be submitted in linuxkpi in base.

Reply:
https://reviews.freebsd.org/D30024